deltachat-ffi/deltachat.h

```diff
@@ -403,6 +403,9 @@ char*           dc_get_blobdir               (const dc_context_t* context);
  *                    to not mess up with non-delivery-reports or read-receipts.
  *                    0=no limit (default).
  *                    Changes affect future messages only.
+ * - `debug_logging` = Let the core save all events to the database. You should expose this as an advanced
```

Comment:
Documentation should also say what are the values and which one is the default, like all options above.